src/pages/Dashboard/Dashboard_Logic.tsx

```diff
@@ -43,13 +60,53 @@ const Dashboard: React.FC = () => {
     return `${_time.toString()} hr`;
   };
 
+  const onMyPlayToggle = () => setMyPlayEnabled(prev => !prev);
+
+  const overallStatsArray = [
+    {
+      name: "Total Liquidity",
+      stat: stats?.tvl
+        ? `$${formatNumberWithCommas(ethers.utils.formatEther(stats.tvl))}`
```

Comment:
Should be `formatUnits` since some tokens use different decimal values (eg USDT)